src/Ganeti/JSON.hs

```diff
@@ -109,7 +109,7 @@ import Text.JSON.Pretty (pp_value)
 -- itself to build many other modules.
 
 import Ganeti.BasicTypes
-import Ganeti.Compat
+import Ganeti.Compat ()
```

Comment:
just for curiosity and for non Haskell programmers: what's the difference between `import something` and `import something ()`